pkg/controllers/subnetport/addressbinding_webhook.go

```diff
@@ -56,8 +57,24 @@ func (v *AddressBindingValidator) Handle(ctx context.Context, req admission.Requ
 			log.Error(err, "error while decoding AddressBinding", "AddressBinding", req.Namespace+"/"+req.Name)
 			return admission.Errored(http.StatusBadRequest, err)
 		}
-		if !reflect.DeepEqual(ab.Spec, oldAddressBinding.Spec) {
-			return admission.Denied("update AddressBinding is not allowed")
+		if ab.Spec.VMName != oldAddressBinding.Spec.VMName || ab.Spec.InterfaceName != oldAddressBinding.Spec.InterfaceName {
+			return admission.Denied("update AddressBinding vmName/interfaceName is not allowed")
+		}
+	}
+	if ab.Spec.IPAddressAllocationName != "" {
+		ipAllocation := &v1alpha1.IPAddressAllocation{}
+		if err := v.Client.Get(context.TODO(), types.NamespacedName{
```

Comment:
I suggested to do validation in the webhook, but I like to understand do we have any performance concern to get IPAddressAllocation CR in the webhook. @lxiaopei ?